.github/workflows/tests.yml

```diff
@@ -52,12 +52,12 @@ jobs:
           wait-on: "http://localhost:3000"
           command: yarn cy:run
           config-file: cypress.config.js
-      - uses: actions/upload-artifact@v3
+      - uses: actions/upload-artifact@v4
```

Comment:
This is the latest major version of `actions/upload-artifact` at the time of this writing.